x-ocaml.opam

```diff
@@ -19,6 +19,7 @@ depends: [
   "ocamlfind" {>= "1.9.8"}
   "ppx_blob" {>= "0.9.0"}
   "odoc" {with-doc}
+  "ppx_deriving_yojson" {>= "3.9.0"}
```

Comment:
This opam file is automatically generated from the `dune-project` file by `dune build`, could you please add the dependency there? :)